crates/blockchain-tree/Cargo.toml

```diff
@@ -19,13 +19,15 @@ reth-interfaces = { path = "../interfaces" }
 reth-db = { path = "../storage/db" }
 reth-provider = { path = "../storage/provider" }
 
+
 # common
 parking_lot = { version = "0.12"}
 lru = "0.10"
 tracing = { workspace = true }
 
 # mics
 aquamarine = "0.3.0"
+linked_hash_set = "0.1.4"
```

Comment:
this one's fine as well, but I specifically referenced the indexmap crate, which should do behave the same I believe.

but since we only expect a few hashes this doesn't really matter, and this crate is already in the dep tree.